File: Library/Python/Library.py
from Book import Book

class Library:

	def __init__(self, branch, city, zipcode):
		self.branch = branch
		self.city = city
		self.zipcode = zipcode
		self.bookshelf = []	
		
	def addBook(self,Book):
		self.bookshelf.append(Book)

	def delBook(title):
		for Book in bookshelf:
			if Book.title == title:
				bookshelf.remove(Book)
		
	def printBooks(self):
		print"These are all the books at the ",self.branch," located at the city of ",self.city,"."
		for x in range(len(self.bookshelf)):
			print self.bookshelf[x].getTitle()
		

File: Library/Java/Library.java
import java.io.*;
import java.util.*;

public class Library extends Book{
	String branch;
	String city;
	int zipcode;
	//int maxSize;
	ArrayList<Book> bookshelf;
	
	Library(){
		
	}
	
	Library(String branch, String city, int zipcode){
		this.branch = branch;
		this.city = city;
		this.zipcode = zipcode;
		//this.maxSize = 6;
		this.bookshelf = new ArrayList<Book>();
	}
	
	public String getBranch() {
		return this.branch;
	}
	
	public String getCity() {
		return this.city;
	}
	
	 public void addBook(Book bd) {
		
		this.bookshelf.add(bd);
		
	}
	 
	 public void printBooks() {
		 System.out.printf("These are the books that are located at the %s branch in the city %s:\n", getBranch(), getCity());
		 for(int i = 0; i < this.bookshelf.size(); i++) {
			 System.out.print(this.bookshelf.get(i).getTitle()+"  \n");
		 }
	 }
}
